app/app/Http/Controllers/SocialAuthController.php

```diff
@@ -42,6 +43,13 @@ private function createNewUserWithGoogle($profile) {
 			$newUser->email = $email;
 			$newUser->first_name = $profile->firstName;
 			$newUser->last_name = $profile->lastName;
+			$newUser->home_zipcode = $profile->zip;
+			$newUser->home_region = $profile->region;
+			$country = $profile->country;
+			if(!empty($country)) {
+				$country_id = Country::where("name","like",$profile->region)->first()->id;
+				$newUser->home_region = $country_id;
```

Comment:
The "home_region" should be matched with something like state/province or territory instead of the country id.

**Replace**:
     $newUser->home_region = $country_id;
**with**:
     $newUser->home_country_id = $country_id.